### ISSUE TITLE
wavelet_masking_ratio is always fixed at 1.0 in anima_train_network.py due to timestep scaling

### ISSUE THREAD

**Issue by Muffalo52** [67372a/sd-scripts#230]
When using the --wavelet_masking option for Latent Wavelet Diffusion during Anima training (anima_train_network.py), the wavelet_mask_ratio is constantly fixed at 1.0. The spatial mask does not function correctly, and the entire target image is uniformly applied during training.

In anima_train_network.py, inside the get_noise_pred_and_target function (as well as get_ileco_noise_pred_and_target and get_addift_noise_pred_and_target), the timesteps variable is scaled down before being passed to the model:

timesteps = timesteps / 1000.0  # scale to [0, 1] range. timesteps is float32

However, this scaled timesteps variable is directly returned to the base NetworkTrainer class and passed to the get_wavelet_mask function. The get_wavelet_mask function expects timesteps to be in the original scale (e.g., 0 ~ 1000). Because the passed timesteps value is now between 0 and 1, the mask calculation logic always evaluates to True, forcing the mask value to be 1.0 for all pixels.

Proposed Fix:
The timesteps scaling should be isolated so that the original timesteps (in the 0 ~ 1000 scale) is returned to the base class, while only the scaled timesteps are passed to the anima model's forward pass.

Example modification in get_noise_pred_and_target:

        # Keep original timesteps intact, create a new variable for the model
        model_timesteps = timesteps / 1000.0  # scale to [0, 1] range.

        # (Skip) ...

        with torch.set_grad_enabled(is_train), accelerator.autocast():
            model_pred = anima(
                noisy_model_input,
                model_timesteps, # <-- Pass the scaled timesteps to the model
                prompt_embeds,
                padding_mask=padding_mask,
                target_input_ids=t5_input_ids,
                target_attention_mask=t5_attn_mask,
                source_attention_mask=attn_mask,
            )

        # (Skip) ...
        
        # Return the original unscaled timesteps
        return model_pred, target, timesteps, weighting, noise

(This same separation should also be applied to get_ileco_noise_pred_and_target and get_addift_noise_pred_and_target in the same file).

<img width="816" height="479" alt="Image" src="https://github.com/user-attachments/assets/f33635f7-ad9c-474b-aa5d-3fd718fc4c47" />
<img width="805" height="468" alt="Image" src="https://github.com/user-attachments/assets/95cfbdaa-0003-40fe-b9df-e7b18aa39e52" />

before and after the fix
